### ISSUE TITLE
Update 3.13.4 fails ...

### ISSUE THREAD

**Issue by jensolafkoch** [vendidero/woocommerce-germanized#192]
... with message:

`Installation fehlgeschlagen: Die Datei konnte nicht kopiert werden. woocommerce-germanized/packages/woocommerce-germanized-dhl/assets/wsdl/OneClickForAppV3?wsdl.wsdl`

I had never any problems before.

Issue persists even after deinstalling the plugin.

Any ideas?

**Comment by dennisnissle** [comment]
Hi there,

I've got an idea on the issue - are you using a windows machine? Seems like windows is rejecting the filename due to the question mark. I've already prepared a patch for it.

Best,
Dennis

**Comment by jensolafkoch** [comment]
Yes, Windows. I also wondered about the question mark. When I looked at the zip itself I saw an underscore though?

**Comment by dennisnissle** [comment]
> When I looked at the zip itself I saw an underscore though?

Really? Maybe that's what windows does (automatically) when extracting the zip locally. I've renamed the files and adjusted the logic a little bit. I'll release a patch update in a few minutes.

**Comment by dennisnissle** [comment]
Just released 3.13.5 - could you please test the update again?

**Comment by jensolafkoch** [comment]
Thanks, does work again! :-)
